Automattic/jetpack
# PHPCS: fix linting errors in the Comic CPT file.

See #21963

#### Changes proposed in this Pull Request:

* Fix linting errors with the Comic CPT.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

**Prerequisites:**

- On WordPress.com Simple: enable the Panel theme.
- On self-hosted: add the following snippet to your theme's `functions.php` file, or to a functionality plugin on your site:
```phpe
// Enable comics.
add_action( 'after_setup_theme', function () {
        add_theme_support( 'jetpack-comic' );
} );
```

In your dashboard, you should now see a new comic menu.

**To test**

* Start by going to the Posts menu, and select a few of your existing posts, possibly with Featured images.
* In the Bulk edit option dropdown, you'll have an option to convert those posts to comics. Do it.
* On page reload, you should see a notice to confim the conversion worked:

<img width="1298" alt="Screenshot 2022-05-02 at 11 06 41" src="https://user-images.githubusercontent.com/426388/166212168-8f90ec7d-b5e5-4c21-a253-dce7ca018e92.png">

* If you access the comic menu, you will see your freshy converted posts there.
* You should be able to convert some of your comics back to posts following the same process.
* Access your site's frontend.
* Drag an image anywhere on the page; it should trigger an upload and create a new comic post for you, with the image in the post body as well as set as the featured image.
* You should automatically be redirected to the post editor for that new comic.
* Check the html view for the classic block with the image; every image attribute should be correct.
* Publish the post and go back to the Frontend.
* Now drag multiple images at once.
* It should create a new post for you, with a gallery inserted in the post content.
